homeassistant/components/ecobee/manifest.json

```diff
@@ -3,6 +3,7 @@
   "name": "ecobee",
   "codeowners": [],
   "config_flow": true,
+  "dependencies": ["http"],
```

Comment:
Hmm wondering why `http` a dependency right now actually?

I think this can be removed.

```suggestion
```